contracts/interfaces/IStrategyBase.sol

```diff
@@ -1,17 +1,43 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.7.0;
 
+/// @title IStrategyBase Interface
+/// @author imToken Labs
 interface IStrategyBase {
+    /// @notice Emitted when spender address is updated
+    /// @param newSpender The address of the new spender
     event UpgradeSpender(address newSpender);
+
+    /// @notice Emitted when allowing another account to spend assets
+    /// @param spender The address that is allowed to transfer tokens
     event AllowTransfer(address indexed spender, address token);
+
+    /// @notice Emitted when disallowing an account to spend assets
+    /// @param spender The address that will be removed from allowing list
```

Comment:
```suggestion
    /// @param spender The address that is removed from allow list
```

Reply:
Adjusted in 7a91187197c37a8285b47ee4f3227bad434ebccb.